code/game/objects/_items.dm

```diff
@@ -1005,4 +1005,7 @@ THIS SCOPE CODE IS DEPRECATED, USE AIM MODES INSTEAD.
 
 
 /obj/item/repair_needed()
-	return max_health - health
\ No newline at end of file
+	return max_health - health
+
+/obj/item/stasis_act()
```

Comment:
There should probably be some vars for intensity and duration, not all status is made the same